app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,14 @@
+class PhysicalServerDecorator < MiqDecorator
+  def img_for_physical_vendor
+    "svg/vendor-#{label_for_vendor.downcase}.svg"
+  end
+
+  def img_for_health_state
```

Comment:
AFAIK decorators don't have this pattern yet, so I'd like to ask the guys from above to check this out to not have a bad pattern that gets copied to other places :wink: